pkg/controller/endpointmonitor/endpointmonitor_controller_test.go

```diff
@@ -43,7 +43,68 @@ var (
 	}
 )
 
-func TestEndpointMonitorReconcile(t *testing.T) {
+func TestEndpointMonitorReconcileCreate(t *testing.T) {
+	controllerConfig := config.GetControllerConfigTest()
```

Comment:
Can the common part of these tests be moved to a helper method ?

Reply:
I tried doing that but they are pretty much dependent on each other. But i'll check again

Reply:
Moved